tasks/main.yml

```diff
@@ -1,27 +1,21 @@
 ---
 - name: Configure the Jenkins APT key
-  apt_key: url=http://pkg.jenkins-ci.org/debian/jenkins-ci.org.key
+  apt_key: url=http://pkg.jenkins.io/debian/jenkins.io.key
            state=present
 
 - name: Configure the Jenkins APT repositories
-  apt_repository: repo="deb http://pkg.jenkins-ci.org/debian binary/"
-                  state=present
+  apt_repository:
+    repo: deb http://pkg.jenkins-ci.org/debian binary/
```

Comment:
```yaml
apt_key: url=http://pkg.jenkins.io/debian/jenkins.io.key
         state=present
```

vs. 

```yaml
apt_repository:
    repo: deb http://pkg.jenkins-ci.org/debian binary/
    state: present
    update_cache: yes
```

Both styles are legit, but the first already exists in this role, so I think we should keep the other tasks consistent.